### PR TITLE
Added media queries for desktop browser

### PR DESCRIPTION
**Pull request by ntazelo**
### In this project

**I added media queries**
**I updated HTML for responsiveness styling**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -382,3 +417,173 @@ body {
...
+    margin-right: auto;
+    margin-left: auto;
+  }
+
+  .cont-btn-form {
+    align-self: center;
+  }
+}
+
+@media (min-width: 900px) {
```
Comment (SajjadAhmad14) [inline style.css:473]: Please write the value for min-width property should in range of 992px - 1100px.

File: style.css [hunk 2]
```diff
@@ -382,3 +417,173 @@ body {
...
+    width: auto;
+  }
+
+  .cont-inp-form {
+    width: 60%;
+    margin: 0 auto;
+  }
+}
+
+@media (min-width: 1200px) {
```
Comment (SajjadAhmad14) [inline style.css:553]: Please write the value for min-width property should in range of 992px - 1100px.

File: style.css [hunk 3]
```diff
@@ -382,3 +417,173 @@ body {
   color: white;
   cursor: pointer;
 }
+
+.cont-menu-list {
+  display: none;
+}
+
+@media (min-width: 600px) {
```
Comment (SajjadAhmad14) [inline style.css:425]: Please write the value for min-width property should in range of 992px - 1100px.

File: style.css [hunk 4]
```diff
@@ -382,3 +417,173 @@ body {
...
+    font-size: 15px;
+    line-height: 20px;
+    color: black;
+  }
+
+  .menu-list > * {
+    margin-right: 1rem;
+  }
+
+  .intro {
```
Comment (SajjadAhmad14) [inline style.css:448]: Please use the font color for the Headline section from the design.

File: index.html [hunk 5]
```diff
@@ -157,163 +172,175 @@ <h2 class="desc__project">Multi-Post Stories</h2>
...
+                <button type="button" class="lng__btn">javascript</button>
+              </li>
+            </ul>
+            <button type="button" class="btn-action">See Project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="about">
```
Comment (SajjadAhmad14) [inline index.html:250]: Please use the font color for the About me section from the design.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Eloi Ntazama, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professiona. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by AshakaE — APPROVED** [review]
### Hi @ntazelo  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the requested changes 👍🏾
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**